templates/zerver/help/invite-new-users.md

```diff
@@ -1,162 +1,90 @@
 # Invite new users
 
-There are a number of ways to grant access to your Zulip organization;
-the article below describes each in more detail.
+When you invite users, you can:
 
-* Allow **anyone to join** without an invitation.
+* Set the [role](/help/roles-and-permissions) that they will have when
+  they join.
 
-* Allow people to join based on the **domain** of their email address.
+* Configure the streams they will automatically be added to.
 
-* Send **email invitations**.
+Your organization may also want to configure default
+[streams](/help/set-default-streams-for-new-users) and
+[settings](/help/configure-default-new-user-settings) for new users.
 
-* Share a **reusable invitation link**.
 
-The last two, invite-based, techniques also allow you to control the
-[role (owner, admin, moderator, member, or guest)](/help/roles-and-permissions) that the
-invited people will have.
+## Send e-mail invitations
 
-You can also manage access by
-[controlling how users authenticate](/help/configure-authentication-methods)
-to Zulip.  For example, you could allow anyone to join without an
-invitation, but require them to authenticate via LDAP.
-
-## Enable email signup
+!!! warn ""
+    You will only see an **Invite users** option if you have permission to
+    invite users to the organization.
 
 {start_tabs}
 
-{tab|restrict-by-email-domain}
-
-1. Set [default streams](/help/set-default-streams-for-new-users) for new users.
-
-{settings_tab|organization-permissions}
-
-1. Find the section **Joining the organization**.
-
-1. Toggle **Invitations are required for joining this organization**.
-
-1. Set **Restrict email domains of new users?** to
-   **Restrict to a list of domains**.
-
-1. Enter any number of domains. For each domain, check or uncheck
-   **Allow subdomains**.
-
-1. Click **Save changes**.
+{relative|gear|invite}
 
-{tab|allow-anyone-to-join}
+1. Enter a list of email addresses.
 
-1. Set [default streams](/help/set-default-streams-for-new-users) for new users.
+1. Select when the invitation will expire.
 
-{settings_tab|organization-permissions}
+1. Select what [role](/help/roles-and-permissions) the users will join as.
 
-1. Find the section **Joining the organization**.
+1. Configure which streams they will be added to.
 
-1. Toggle **Invitations are required for joining this organization**.
+1. Click **Invite**.
 
-1. Set **Restrict email domains of new users?** to either
-   **Don't allow disposable email addresses** (recommended)
-   or **No restrictions**.
+!!! warn ""
 
-1. Click **Save changes**.
+    **Note**: The number of email invites you can send in a day is
+    limited on the Zulip Cloud Free plan. [Contact support](/help/contact-support)
+    if you hit the limit and want to invite more users.
 
 {end_tabs}
 
-Before anyone joins your organization this way, we'll send a validation link
-to verify their email address.
-
-## Send invitations
+## Send an invitation link
```

Comment:
I think this heading, which is for the reusable invitation links, should probably be "Create an invitation link", since Zulip does not send it anywhere for you.

Reply:
Works for me.

Reply:
OK, applied and updated incoming links.